cli_tools/google-osconfig-agent/packaging/debian/google-osconfig-agent.lintian-overrides

```diff
@@ -0,0 +1 @@
+google-osconfig-agent binary: statically-linked-binary
```

Comment:
lintian overrides almost always indicate you're doing the wrong thing.. why must this be statically compiled?

Reply:
This is an artifact of lintians logic for go packages. Lintian is supposed to skip this check for go packages but the way it skips is checking for the build-depends of golang-go or starts with golang-, since we don't depend on that package (salsa go-team package, part of their build toolchain) we get the error.

https://salsa.debian.org/lintian/lintian/blob/master/checks/binaries.pm#L139